apps/docs/src/.vitepress/components/DesignTokenHeader.vue

```diff
@@ -17,19 +18,19 @@ const emit = defineEmits<{
 </script>
 
 <template>
-  <div class="header">
+  <div class="header vp-raw">
```

Comment:
the vp-raw seems to have the effect that now all the preview__areas are 2px bigger - is this intended?

![image](https://github.com/SchwarzIT/onyx/assets/151019977/0f3f9b17-1bfc-46db-83d8-df4230c5e111)

before, the overall size of the orange colored area was e.g. 0.5rem, now only the light-orange space is 0.5rem + 1px border top and bottom

Reply:
Nope, its fixed